lib/trento_web/controllers/about_controller.ex

```diff
@@ -3,15 +3,11 @@ defmodule TrentoWeb.AboutController do
 
   alias Trento.Hosts
 
-  @version Mix.Project.config()[:version]
-  # TODO determine Flavor
-  @flavor "Community"
-
   @spec info(Plug.Conn.t(), map) :: Plug.Conn.t()
   def info(conn, _) do
     json(conn, %{
-      flavor: @flavor,
-      version: @version,
+      flavor: Trento.Installation.flavor(),
+      version: Mix.Project.config()[:version],
```

Comment:
this needs to stay as a module attribute since the release does not have Mix, and we need to inject the version at compile time

Reply:
Fixed.